app/src/main/java/net/sourceforge/opencamera/OpenCameraApplication.java

```diff
@@ -23,12 +17,7 @@ public void onCreate() {
             Log.d(TAG, "onCreate");
         super.onCreate();
         checkAppReplacingState();
-
-        Core core = new Core();
-        if( Build.VERSION.SDK_INT >= Build.VERSION_CODES.O ) {
-            core.createUploadNotificationChannel();
-        }
-        core.initializeNotificationConfig();
+        new Core(this);
```

Comment:
If you dont store the object it will be erased by gc! Add class field and save core there